### PR TITLE
document and example for moveElementBy

### PR DESCRIPTION
**Pull request by OsugaHiroshi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,24 @@ console.log(moveElementByValue(array, 'A', 'C'));  // -> ['B', 'A', 'C'] (Insert
 console.log(moveElementByValue(array, 'A', 'C', true));  // -> ['B', 'C', 'A'] (Insert 'A' after 'C')
 console.log(moveElementByValue(array, 'D', 'C'));  // -> Error! ('D' is not included)
 ```
+
+### `moveElementBy`
+
+```js
+const {moveElementBy} = require('..');
```
Comment (kjirou) [inline README.md]: Please replace `require('..');` to the following code:

```
require('sort-array-for-drag-and-drop');
```

**Review by kjirou — COMMENTED** [review]
@OsugaHiroshi I requested only one modification!

File: README.md (hunk 1 above)
Comment (OsugaHiroshi) [reply README.md]: This `require('..');` is the same to the soruce codes in examples directory.
Should I fix these codes too ?

Comment (OsugaHiroshi) [reply README.md]: I fixed this

**Comment by kjirou** [comment]
Thanks!

**Pull request merged**